core/src/main/scala/cats/data/Prod.scala

```diff
@@ -0,0 +1,97 @@
+package cats
+package data
+
+/**
+ * [[Prod]] is a product to two independent functor values.
+ *
+ * See: [[https://www.cs.ox.ac.uk/jeremy.gibbons/publications/iterator.pdf The Essence of the Iterator Pattern]]
+ */
+final case class Prod[F[_], G[_], A](first: F[A], second: G[A]) extends Serializable
```

Comment:
I don't think there's any harm in it, but since this is a case class, you shouldn't need to explicitly extend `Serializable`.